lib/zendesk_apps_tools/command.rb

```diff
@@ -191,13 +191,69 @@ def update
       deploy_app(:put, "/api/v2/apps/#{app_id}.json", {})
     end
 
+    desc 'migrate', 'Helps with the migration of a v1 app to v2'
+    shared_options(except: [:unattended, :clean])
+    method_option %s(replace-v1), default: false, required: false, type: :boolean, aliases: '-r'
+    method_option :auto, default: false, required: false, type: :boolean, aliases: '-a'
+    def migrate
+      cache.clear
+      setup_path(options[:path])
+      @command = 'Migrate'
+      unless migration_helper_installed
+        try_install = get_value_from_stdin("The Zendesk App Migration Helper isn't installed yet. Would you like to try installing now?", limited_to: ['y', 'yes', 'n', 'no'], default: 'y' )
+        say_error_and_exit("Please install the Zendesk App Migration Helper before running this command") if (try_install =~ /^y(es)?$/).nil? # thats a no
+        say_error_and_exit("Unable to install the Zendesk App Migration Helper \
```

Comment:
nitpick: I think it would read nicer if it was `install_migration_helper` then checking `migration_helper_installed` below.